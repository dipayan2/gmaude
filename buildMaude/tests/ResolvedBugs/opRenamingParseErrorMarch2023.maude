***
***	Test that an op renaming using the sort "to" parses correctly.
***

fmod FOO is
  sorts Foo to .
  op f : Foo to -> Foo .
endfm

fmod BAR is
  inc FOO * (op f : Foo to -> Foo to g) .
endfm

show desugared .
