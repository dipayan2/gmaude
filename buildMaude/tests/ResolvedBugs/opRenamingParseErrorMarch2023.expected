fmod BAR is
  including BOOL .
  including FOO * (op f : [Foo] [to] -> [Foo] to g) .
endfm
Bye.
